src/views/v2/top-bar/today.php

```diff
@@ -18,7 +18,7 @@
 		<?php esc_html_e( 'Today', 'the-events-calendar' ); ?>
 	</a>
 
-	<span class="tribe-common-h3 tribe-common-h3--alt">
+	<span class="tribe-common-h3 tribe-common-h3--alt tribe-events__top-bar-today--title">
 		<?php esc_html_e( 'Now', 'the-events-calendar' ); ?> &mdash; <?php echo date( 'F jS, Y', time() ); ?>
```

Comment:
We should wrap the date in this string in a `<time datetime="..."></time>` for semantic goodness 😃

Reply:
@juanfra